.github/workflows/kit-bootstrap.yml

```diff
@@ -0,0 +1,152 @@
+name: This workflow automates the build, testing, and bootstrapping of a test environment on Kubernetes using KIT CLI
+on:
+  workflow_run:
+    workflows: [trigger-kit-CI]
+    types: [completed]
+permissions:
+  id-token: write
+  pull-requests: write
+  contents: read
+jobs:
+  kit:
+    strategy:
+      matrix:
+        os: [ macos-latest ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/install-go-and-dependencies
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::953421922360:role/kit-Github-workflow
+          role-session-name: githubworkflowsession
+          aws-region: us-west-2
+      - name: Install kubectl
+        run: |
+          if [[ "${{ matrix.os }}" == "ubuntu-latest" ]]; then
+            curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -L -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
+          elif [[ "${{ matrix.os }}" == "macos-latest" ]]; then
+            curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -L -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl"  
+          fi
+          chmod +x kubectl
+          sudo mv kubectl /usr/local/bin/
+      - name: Builds the KIT CLI ,Provisions the Kit Environment,KIT guest cluster and deletes.
+        id: step1
+        run: |
+          cd ${HOME}/work/kubernetes-iteration-toolkit/kubernetes-iteration-toolkit/substrate/cmd/kitctl
+          go build  .
+          if [[ "${{ matrix.os }}" == "ubuntu-latest" ]]; then
+            os_type="linux"    
+          elif [[ "${{ matrix.os }}" == "macos-latest" ]]; then
+            os_type="darwin"
+          fi 
+          ./kitctl bootstrap kit-github-${os_type}-${{ github.run_id }}-${{github.run_attempt}}
+          export KUBECONFIG=${HOME}/.kit/env/kit-github-${os_type}-${{ github.run_id }}-${{github.run_attempt}}/etc/kubernetes/admin.conf
+          kubectl get pods -A
+          for i in 21 22 23 24 25; do
```

Comment:
I believe I have talked about this offline.
I'd suggest defining an array like `allVersions=("1.21" "1.22" "1.23" "1.24" "1.25")` at the very top of this script so that it's more visible.
Then you can have something like `for version in ${allVersions[@]}`.
This help a lot with code readability.